scripts/build-packages.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+cd ./packages/scratch-svg-renderer
+npm run build
+cd -
```

Comment:
No change necessary, and I'm not sure why I didn't think of this last time around, but this could be further simplified to:
```suggestion
npm -C ./packages/scratch-svg-renderer run build
```